core/indigo-core/molecule/src/molecule_json_loader.cpp

```diff
@@ -1115,6 +1115,22 @@ void MoleculeJsonLoader::fillXBondsAndBrackets(Superatom& sa, BaseMolecule& mol)
     }
 }
 
+std::string MoleculeJsonLoader::monomerTemplateClass(const rapidjson::Value& monomer_template)
+{
+    std::string result;
+    if (monomer_template.HasMember("class"))
+        result = monomer_template["class"].GetString();
+    else if (monomer_template.HasMember("classHELM"))
+    {
+        result = monomer_template["classHELM"].GetString();
```

Comment:
add other HELM classes
add tests for other classes

Reply:
tests added